File: src/main/java/com/adamkali/simpleide/editor/lang/tokens/operator/OperatorToken.java
package com.adamkali.simpleide.editor.lang.tokens.operator;

import com.adamkali.simpleide.editor.lang.tokens.Token;
import com.adamkali.simpleide.preferences.EditorColors;

public class OperatorToken extends Token {
    public OperatorToken(String value) {
        super(EditorColors.OPERATOR_BG_COLOR, EditorColors.OPERATOR_FG_COLOR, value);
    }
}

File: src/main/java/com/adamkali/simpleide/project/lang/tokens/operator/OperatorToken.kt
package com.adamkali.simpleide.project.lang.tokens.operator

import com.adamkali.simpleide.preferences.EditorColors
import com.adamkali.simpleide.project.lang.tokens.Token

abstract class OperatorToken(text: String) :
    Token(text, EditorColors.PLAINTEXT_BG_COLOR, EditorColors.PLAINTEXT_FG_COLOR) {
}